i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/tubemq/error_code.md

```diff
@@ -5,33 +5,35 @@ title: 错误码定义
-| 客户端错误| 403          | FORBIDDEN                         | 操作的Topic不存在，或者已删除                          | 需要与管理员确认具体问题原因                   |
-| 客户端错误| 404          | NOT_FOUND                         | 消费offset已经达到最大位置                            |                                           |
-| 客户端错误| 405          | ALL_PARTITION_FROZEN              | 所有可用分区都被冻结                                  |  可用分区已被客户端冻结，需要解冻处理或者等待一段时间再重试  |
-| 客户端错误| 406          | NO_PARTITION_ASSIGNED              | 当前客户端没有被分配分区进行消费                       |  客户端个数超过分区个数，或者服务器还没有进行负载均衡操作，需要等待并重试 |
-| 客户端错误| 407          | ALL_PARTITION_WAITING              | 当前可用分区都达到了最大消费位置                       |  需要等待再重试                              |
-| 客户端错误| 408          | ALL_PARTITION_INUSE                | 当前可用分区都被业务使用未释放                         |  需要等待业务逻辑调用confirm接口释放分区，需要等待并再重试 |
-| 客户端错误| 410          | PARTITION_OCCUPIED                | 分区消费冲突，忽略即可                                 | 内部注册的临时状态，业务接口一般不会遇到该报错            |
-| 客户端错误| 411          | HB_NO_NODE                        | 节点超时，需要降低操作等待一阵后再重试处理                 | 一般出现在客户端在服务器侧心跳超时，这个时候需要降低操作频率，等待一阵待lib注册成功后再重试处理 | 
-| 客户端错误| 412          | DUPLICATE_PARTITION               | 分区消费冲突，忽略即可                                 | 一般是由于节点超时引起，重试即可                             |
-| 客户端错误| 415          | CERTIFICATE_FAILURE               | 认证失败,包括用户身份认证,以及操作授权不通过               | 一般是用户名密码不一致,或者操作的范围未授权,需要结合错误详情进行排查 |
-| 客户端错误| 419          | SERVER_RECEIVE_OVERFLOW           | 服务器接收overflow，需要重试处理                        | 如果长期的overflow，需要联系管理员扩容存储实例，或者扩大内存缓存大小 |
-| 客户端错误| 450          | CONSUME_GROUP_FORBIDDEN           | 消费组被纳入黑名单                                     | 联系管理员处理                                               |
-| 客户端错误| 452          | SERVER_CONSUME_SPEED_LIMIT        | 消费被限速                                            | 联系管理员处理，解除限速                                     | 
-| 客户端错误| 455          | CONSUME_CONTENT_FORBIDDEN         | 消费内容拒绝，包括消费组禁止过滤消费，过滤的数据流ID集合与允许的ID集合不一致等 | 对齐过滤消费的设置确认没有问题后，再联系管理员处理   |
-| 服务器侧异常 | 500       | INTERNAL_SERVER_ERROR             | 内部服务器错误                                         | 需要结合错误信息，联系管理员确定问题原因后重试 |
-| 服务器侧异常| 503        | SERVICE_UNAVILABLE                | 业务临时禁读或者禁写                                    | 继续重试处理，如果持续的出现该类错误，需要联系管理员处理     |
-| 服务器侧异常| 510        | INTERNAL_SERVER_ERROR_MSGSET_NULL | 读取不到消息集合                                       | 继续重试处理，如果持续的出现该类错误，需要联系管理员处理     |
+| 错误类别     | 错误码 |      错误标记                     |            含义                                    |              备注                          |
+| ---------- |-----| -------------------------------- | ------------------------------------------------- | ----------------------------------------- |
+| 成功操作    | 0   | SUCCESS                           | 操作成功                                            |                                          |
```

Comment:
0 or 200 are all successful, errcode 200 needs to reserved and 0 needs to be supplemented.